packages/graphql-server/src/globalContext.ts

```diff
@@ -40,6 +40,13 @@ export const createContextProxy = () => {
         const ctx = store?.get('context') || {}
         return ctx[property]
       },
+      set: (_target, property: string, newVal) => {
```

Comment:
@twodotsmax I'm wondering if its possible to reuse the `setContext` function here instead of repeating the code?